opencog/timeoctomap/TimeOctomap.cc

```diff
@@ -29,67 +29,71 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-//TimeSpaceAtom.cpp
-#include "TimeSpaceAtom.h"
+//TimeOctomap.cpp
+#include "TimeOctomap.h"
 //#include "octomap/OcTreeKey.h"
 //#include <assert.h>
 #include "opencog/util/oc_assert.h"
-TimeSpaceAtom::TimeSpaceAtom(unsigned int num_time_units,
-                             vector<double>map_res_meters):
-    time_circle(num_time_units), created_once(false)
+
+TimeOctomap::TimeOctomap(unsigned int num_time_units,
+                         double map_res_meters,
+                         duration_c time_resolution):
+    time_circle(num_time_units),map_res(map_res_meters),time_res(time_resolution),created_once(false)
```

Comment:
Hi Nil,
 I am confused. I use vim, however some places are supposed to be tabs and
some spaces. I am a very basic user of vim, i don't know advanced stuff. If
it were possible to automatically indent the right way, then can you let me
know?
Regards,
Mandeep

On Thu, Jun 16, 2016 at 2:02 PM, ngeiswei <notifications@github.com> wrote:

> In opencog/timeoctomap/TimeOctomap.cc
> <https://github.com/opencog/opencog/pull/2196#discussion_r67291093>:
>
> >  //#include "octomap/OcTreeKey.h"
> >  //#include <assert.h>
> >  #include "opencog/util/oc_assert.h"
> > -TimeSpaceAtom::TimeSpaceAtom(unsigned int num_time_units,
> > -                             vector<double>map_res_meters):
> > -    time_circle(num_time_units), created_once(false)
> > +
> > +TimeOctomap::TimeOctomap(unsigned int num_time_units,
> > +                         double map_res_meters,
> > +                         duration_c time_resolution):
> > +    time_circle(num_time_units),map_res(map_res_meters),time_res(time_resolution),created_once(false)
>
> To avoid some gdb warning, time_circle should be initialized after map_res
> and time_res.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/opencog/opencog/pull/2196/files/97b3d2f5a4be7058b52da07b66fbe4fdd15d3f28#r67291093>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHM_f31hEXEylPsSZURTpsR9G0mXd2k6ks5qMObcgaJpZM4I2AqA>
> .
>